App/src/main/java/cc/softwarefactory/lokki/android/Dialogs.java

```diff
@@ -24,75 +25,46 @@
 
     public static void addPeople(final Context context) {
 
-        AlertDialog.Builder alertDialog = new AlertDialog.Builder(context)
-                .setTitle(context.getResources().getString(R.string.add_contact))
-                .setMessage(context.getResources().getString(R.string.add_contact_dialog_message))
-                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        dialog.dismiss();
-                    }
-                });
-        alertDialog.show();
+        showDialog(context, R.string.add_contact, R.string.add_contact_dialog_message);
     }
 
     public static void addPeopleSave(final Context context, Set<String> emails) {
 
         Log.e(TAG, "emails: " + emails);
-        AlertDialog.Builder alertDialog = new AlertDialog.Builder(context);
-        alertDialog.setTitle(context.getResources().getString(R.string.add_contact));
-        String message = "";
-        for (String email : emails) {
-            message += Utils.getNameFromEmail(context, email);
-            if (emails.size() > 1) {
-                message += ", ";
-            } else {
-                message += " ";
-            }
-        }
-        message += context.getResources().getString(R.string.add_contact_dialog_save);
-        alertDialog.setMessage(message)
-                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        dialog.dismiss();
-                    }
-                });
-        alertDialog.show();
+        String title = context.getResources().getString(R.string.add_contact);
+        String message = TextUtils.join(", ", emails) + context.getResources().getString(R.string.add_contact_dialog_save);
```

Comment:
You could have a placeholder in localization instead of concatenating strings here